Link property

josejavierfm opened the pull request:
Link with linkname and linkblank properties asociated

josejavierfm committed:
Add files via upload

ahora puede tener enlace

josejavierfm committed:
Ahora puede tener enlace ademas de texto

josejavierfm committed:
nueva propiedad, linkblank

correccion error y nueva propiedad para el enlace, si abre o no en ventana nueva

richardhsu reviewed (changes requested):
Thanks for contributing, again this project isn't really in any active development but this addition looks good, please review my comments and update the PR and we can get it merged, thanks!

josejavierfm committed:
property linkColor & changes suggested by creator

property linkColor and changes suggested by creator

josejavierfm committed:
property linkColor & changes suggested by creator

property linkColor & changes suggested by creator

josejavierfm committed:
property linkColor & changes suggested by creator

property linkColor & changes suggested by creator
include testLink.html to test it

josejavierfm committed:
reindent file

josejavierfm commented:
i see the file weel indent on my IDE now

richardhsu commented:
@josejavierfm Thank you, merged!
